### ISSUE TITLE
Performance on COCO

### ISSUE THREAD

**Issue by mboudiaf** [Yang-Bob/PMMs#14]
Hi Bob,

I'm trying to reproduce results on COCO, but the results I obtain stand almost 3% below the ones announced in the paper. Here is what I got : 

| Coco 1-shot | Fold-0 | Fold-1 | Fold-2 | Fold-3 | Mean |
|-------------|--------|--------|--------|--------|------|
| Paper       | 29.5   | 36.8   | 28.9   | 27.0   | 30.6 |
| 1-shot      | 27.1   | 31.6   | 26.5   | 26.8   | 28.0 |

For info, I used this repo and followed scrupulously the same training procedure as the one mentioned in the paper. I got the best models respectively at epochs : 55k, 115k, 60k, 110k 
Would it be possible to release your pre-trained models for COCO to assess whether I have an error in my reproduction ? Thank you very much.

Malik

**Comment by Yang-Bob** [comment]
Hi Malik,

I have just checked my Github last weekend and found your question.
The inference on coco may be random since the random seed in coco_val.py is not set. You can set the random seed e.g. `np.random.seed(1385)` to constrain it. 
At last, the **PMMs** model trained by ourselves has been released on [https://github.com/Yang-Bob/PMMs_COCO_parameter](https://github.com/Yang-Bob/PMMs_COCO_parameter), which you can compare with. 

Bob

**Comment by mboudiaf** [comment]
Hi Bob,

Thank you very much for your answer and for the models :)

Malik
